### PR TITLE
Implemented additional check (chapter 5) and corrected an issue displaying an image (chapter 1)

### PR DESCRIPTION
**Pull request by kgronau**
# 11402 Patch me if you can

# General requirements
1. Information in the workshop is adequate and updated Y
2. Code is correct and working Y
3. Links are correct Y
4. Make sure you enter the help email link in your manifest.json Y
5. Please make sure WMS URLs are updated as needed after your PR is approved Y

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x ] All filenames are lower case (including folders, images, files, etc.)
- [x ] Filenames are descriptive
- [ x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by klazarz — APPROVED** [review]

**Review by kaylien — DISMISSED** [review]

**Comment by kaylien** [comment]
@kgronau 

Please click the "Resolve conflicts" button on this page and get rid of the conflicts that are blocking the merge. Once done, we will approve your PR as quickly as we can. Thanks!

**Review by kaylien — APPROVED** [review]

**Pull request merged**
